packages/api-cardano-db-hasura/src/CardanoNodeClient.ts

```diff
@@ -15,6 +17,17 @@ export type LedgerState = {
   }
 }
 
+const fileTypeFromEra = (era: string) => {
+  switch (era) {
+    case 'shelley' :
+      return 'TxSignedShelley'
+    case 'allegra' :
+      return 'TxSignedShelley'
```

Comment:
Shouldn't it be TxSignedAllegra ?

Reply:
Ah yes, this was a misunderstanding of the `type`